Content.Server/Salvage/SalvageSystem.Expeditions.cs

```diff
@@ -76,6 +77,14 @@ private void OnExpeditionShutdown(EntityUid uid, SalvageExpeditionComponent comp
     {
         component.Stream = _audio.Stop(component.Stream);
 
+        // First wipe any disks referencing us
+        var disks = EntityQueryEnumerator<ShuttleDestinationCoordinatesComponent>();
```

Comment:
```suggestion
        var disks = AllEntityQuery<ShuttleDestinationCoordinatesComponent>();
```
I think we need to do this for paused entities as well. For example if someone has a disk in their pocket and gets polymorphed.

Reply:
Ah that’s true; good catch.